docs/pages/guides/environment-variables.mdx

```diff
@@ -59,16 +61,31 @@ If you create an environment-specific file, like **.env.test**, you can load it
 
 <Terminal cmd={['$ NODE_ENV=production npx expo start']} />
 
+### Disabling environment variables
+
+Environment variables in Expo CLI have two parts, both can be disabled:
+
+1. Expo CLI will load the `.env` files into the global process. This can be disabled by setting the environment variable `EXPO_NO_DOTENV` before any Expo CLI command is run, like `EXPO_NO_DOTENV=1`.
+2. Expo's Metro config will inline environment variables in the client JavaScript bundle during serialization. This can be disabled with `EXPO_NO_CLIENT_ENV_VARS`, like `EXPO_NO_CLIENT_ENV_VARS=1`.
+
+If you're experiencing issues with environment variables, you can try disabling one or both of these features. Consider contacting support on GitHub or Discord if you're still having issues.
```

Comment:
```suggestion
If you're experiencing issues with environment variables, you can try disabling one or both of these features.
```

There is no need to explicitly mention where to contact support. Every page in docs explicitly provide link to Discord, forums, etc.